rocAL/rocAL_pybind/amd/rocal/plugin/generic.py

```diff
@@ -22,66 +22,79 @@
 import numpy as np
 import rocal_pybind as b
 import amd.rocal.types as types
-import ctypes
+
 
 class ROCALGenericIterator(object):
-    def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = False, multiplier = [1.0,1.0,1.0], offset = [0.0, 0.0, 0.0], tensor_dtype=types.FLOAT, display=False, device="cpu", device_id =0):
+    def __init__(self, pipeline, tensor_layout=types.NCHW, reverse_channels=False, multiplier=[1.0, 1.0, 1.0], offset=[0.0, 0.0, 0.0], tensor_dtype=types.FLOAT, display=False, device="cpu", device_id=0):
         self.loader = pipeline
```

Comment:
Why spacing is adjusted here?

Reply:
In google python coding style, function arguments has no space between =, so this was changed